exercises/concept/election-day/.docs/instructions.md

```diff
@@ -51,11 +50,8 @@ Create a function `IncrementVoteCount` that will take a counter (`*int`) as an a
 var votes int
 votes = 3
 
-var voteCounter *int
-voteCounter = &votes
-
-IncrementVoteCount(voteCounter, 2)
-*voteCounter == 5 // true
+IncrementVoteCount(&votes, 2)
+votes == 5 // true
```

Comment:
```suggestion
var voteCounter *int
voteCounter = &votes

IncrementVoteCount(voteCounter, 2)

votes == 5          // true
*voteCounter == 5   // true
```

I think being more explicit about the types is better. I did like the fact you showed that the variable was changed with `votes == 5`, so I think we should keep that.